libraries/ashrae/guideline36/5.16.14-multiple-zone-vav-ahu-afdd.ttl

```diff
@@ -0,0 +1,430 @@
+    sh:property [
+        rdfs:label "SAT_AVG" ;
+        rdfs:comment "rolling average of supply air temperature" ;
+        sh:path brick:hasPoint ;
+        sh:qualifiedValueShape [ sh:class brick:Supply_Air_Temperature_Sensor ] ;
+        sh:qualifiedMinCount 1 ;
+    ] ;
+    # sh:property [
+    #     rdfs:label "Delta_T_SF" ;
+    #     rdfs:comment "Temperature rise across supply fan" ;
+    #     sh:path brick:hasPoint ;
+    #     TODO missing from brick
+    #     sh:qualifiedValueShape [ sh:class brick:Air_Differential_Temperature_Sensor ] ;
+    #     sh:qualifiedMinCount 1 ;
+    # ] ;
+    sh:property [
+        rdfs:label "MAT_AVG" ;
+        rdfs:comment "rolling average of mixed air temperature" ;
+        sh:path brick:hasPoint ;
+        sh:qualifiedValueShape [ sh:class brick:Mixed_Air_Temperature_Sensor ] ;
+        sh:qualifiedMinCount 1 ;
+    ] .
+
+:fc-13 a sh:NodeShape, owl:Class, bmotif:Analytics_Application ;
+    bmotif:domain bmotif:HVAC ;
+    rdfs:label "FC#13" ;
+    rdfs:comment "SAT too high in full cooling" ;
+    sh:class brick:Air_Handling_Unit ;
+    sh:property [
+        rdfs:label "SAT_AVG" ;
+        rdfs:comment "rolling average of supply air temperature" ;
+        sh:path brick:hasPoint ;
+        sh:qualifiedValueShape [ sh:class brick:Supply_Air_Temperature_Sensor ] ;
+        sh:qualifiedMinCount 1 ;
+    ] ;
+    sh:property [
+        rdfs:label "SATSP" ;
+        rdfs:comment "supply air temperature setpoint" ;
+        sh:path brick:hasPoint ;
+        sh:qualifiedValueShape [ sh:class brick:Supply_Air_Temperature_Setpoint ] ;
+        sh:qualifiedMinCount 1 ;
+    ] ;
+    sh:property [
+        sh:path brick:hasPart ;
+        sh:qualifiedValueShape [
+            sh:class brick:Chilled_Water_Coil ;
+            sh:property [
+                rdfs:label "CC" ;
+                rdfs:comment "cooling-coil valve position command; 0% <= CC <= 100%" ;
+                sh:path brick:hasPoint ;
+                sh:qualifiedValueShape [ sh:class brick:Valve_Position_Command ] ;
+                sh:qualifiedMinCount 1 ;
+                sh:qualifiedMaxCount 1 ;
+            ] ;
+        ] ;
+        sh:qualifiedMinCount 1 ;
+    ] .
+
+:fc-14 a sh:NodeShape, owl:Class, bmotif:Analytics_Application ;
+    bmotif:domain bmotif:HVAC ;
+    rdfs:label "FC#14" ;
+    rdfs:comment "Temperature drop across inactive cooling coil" ;
+    sh:class brick:Air_Handling_Unit ;
+    sh:property [
+        rdfs:label "CCET_AVG" ;
+        rdfs:comment "rolling average of cooling-coil entering temperature (Depending on the AHU configuration, this could be the MAT or a separate sensor for this specific purpose.)" ;
+        sh:path brick:hasPoint ;
+        sh:qualifiedValueShape [
+        # TODO entering air temp sensor isn't available in brick
+        #     sh:or (
+        #         sh:class brick:Entering_Air_Temperature_Sensor?
+                sh:class brick:Mixed_Air_Temperature_Sensor
```

Comment:
@gtfierro FYI.

Reply:
I agree with using mixed air temp sensor for now. In the future, we may want to use 223P to model the "entering" air temp sensor for the coil